pkgs/development/tools/devcontainers-cli/default.nix

```
@@ -0,0 +1,45 @@
+{
+  lib,
+  stdenv,
+  fetchzip,
+  nodejs_18
+}:
+let
+  pname = "devcontainers-cli";
+  version = "0.54.1";
+  hash = "sha256-L6sVDmKuFmrf9Bm9M54ABmmWFVB8ZlqU+5gAeITBS1Q=";
+in
+stdenv.mkDerivation {
+  inherit pname version;
+
+  src = fetchzip {
+    inherit hash;
+    url = "https://registry.npmjs.org/@devcontainers/cli/-/cli-${version}.tgz";
+  };
+
+  dontBuild = true;
+  dontConfigure = true;
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp -a "$src/." "$out"
+    rm devcontainer.js
+  '';
+
+  postFixup = ''
+    cat <<EOF > $out/bin/devcontainer
+    #!${nodejs_18}/bin/node
+      require('$out/dist/spec-node/devContainersSpecCLI');
+    EOF
+
+    chmod +x $out/bin/devcontainer
```

The program doesn't work properly unless you have git and docker installed.

```console
[nix-shell:~/git/example-devcontainer-project]$ devcontainer build --workspace-folder .
[6 ms] @devcontainers/cli 0.54.1. Node.js v18.18.2. linux 6.1.55 x64.
[82 ms] Failed to fetch control manifest: unable to get local issuer certificate
Error: spawn docker ENOENT
    at ChildProcess._handle.onexit (node:internal/child_process:284:19)
    at onErrorNT (node:internal/child_process:477:16)
    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)
{"outcome":"error","message":"spawn docker ENOENT","description":"An error occurred building the container."}
```

Also, we could probably shorten it down with a `makeWrapper` like @drupol suggested. Something like this?

```suggestion
    makeWrapper "${nodejs_18}/bin/nodejs" "$out/bin/devcontainer" \
      --add-flags "$out/dist/spec-node/devContainersSpecCLI" \
      --prefix PATH : ${lib.makeBinPath [ git docker ]}
```